src/plugins/kibana/public/dashboard/index.js

```diff
@@ -239,7 +241,8 @@ app.directive('dashboardApp', function (Notifier, courier, AppState, timefilter,
         ui: $state.options,
         save: $scope.save,
         addVis: $scope.addVis,
-        addSearch: $scope.addSearch
+        addSearch: $scope.addSearch,
+        timefilter: $scope.timefilter
```

Comment:
This should be needed, the timepicker should just interact with the timefilter service when used in the context of the chrome. Of course the time picker should remain reusable since its only real purpose is it set properties of an object.